public/stylesheets/game-detail.css

```diff
@@ -166,19 +170,24 @@ button {
 
 .game-comment-user {
   padding-right: 10px;
+  font-weight: bolder;
+  text-decoration: underline;
 }
 
 #comment_box {
   width: 400px;
   height: 50px;
+  margin-top: 10px;
+  margin-bottom: 5px;
+  padding: 2%;
 }
 
 .status-form {
   display: flex;
   justify-content: center;
-  margin: 1em;
+  /* margin: 1em; */
   padding-top: 7px;
-  padding-bottom: 20px;
+  padding-bottom: 3rem;
```

Comment:
is this supposed to be rem and not em?

Reply:
Either work in this case - rem is just in relation to the root font size rather than parent element font size I believe. This just matches the blank space between the buttons and the bottom of the container with the game title and the top of the container.